en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -29,34 +47,54 @@ Common examples of hidden folders that will not be synced include `.vscode`, `
 
 ### Sync settings are not synced
 
-Sync settings are not synced so that each each device can be separately configured according to your needs.
-
-> [!tip] Restart the app to apply settings changes on other devices
-> Obsidian **only** applies vault settings when the app starts. If you change a setting on one device, you **must restart** Obsidian on your other devices for the new changes to take effect. 
-> 
-> For example, if you change the path of your daily notes in the [[Daily Notes]] plugin, you need to restart Obsidian on your other devices to use the new path.
+Sync settings themselves are not synced so that each each device can be separately configured according to your needs.
 
 ## Updating your synced vault settings
 
-If you have an existing multi-device sync setup and want to modify your sync settings across devices.
+If you need to modify your sync settings across multiple devices, follow these steps:
+
+> [!tip] In this context, the remote vault does not differentiate between primary and secondary devices; these terms are used for clarity.
 
 ### Primary device
 
-The primary device is your source-of-truth device. It can also mean the device you make the new change on, that you wish to sync across your other devices.
+The primary device is considered your source-of-truth. This is the device on which you make changes that you want to sync across all other devices.
 
 1. Access **Settings** → **Sync**.
 2. Activate the desired settings under **Vault configuration sync**.
-3. Restart Obsidian. On mobile or tablet, this may require a force quit.
+3. Reload or Restart Obsidian. On mobile or tablet, this may require a force quit.
 4. Allow time for the settings to synchronize with your remote vault.
 
 ### Secondary device(s)
 
-The secondary device is another device you use Obsidian with (e.g. your phone). It can also mean the devices you wish to receive an updated change on.
+The secondary device is any other device you use with Obsidian, such as your phone. It refers to devices that receive updates from the primary device.
 
 1. Navigate to **Settings** → **Sync**.
 2. Enable the desired settings under **Vault configuration sync**.
 3. Wait for the changes to be downloaded from your remote vault.
-4. Restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
+4. Reload or Restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
```

Comment:
```suggestion
4. Reload or restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
```